packages/sitecore-jss-nextjs/src/middleware/redirects-middleware.ts

```diff
@@ -9,13 +9,16 @@ import {
   SiteInfo,
 } from '@sitecore-jss/sitecore-jss/site';
 import { getPermutations } from '@sitecore-jss/sitecore-jss/utils';
+import { NextURL } from 'next/dist/server/web/next-url';
 import { NextRequest, NextResponse } from 'next/server';
 import regexParser from 'regex-parser';
 import { MiddlewareBase, MiddlewareBaseConfig } from './middleware';
-import { NextURL } from 'next/dist/server/web/next-url';
 
 const REGEXP_CONTEXT_SITE_LANG = new RegExp(/\$siteLang/, 'i');
 const REGEXP_ABSOLUTE_URL = new RegExp('^(?:[a-z]+:)?//', 'i');
+const NAME_NETLIFY = 'netlify';
```

Comment:
In case you leave netlify check (see my question below)
```suggestion
const NETLIFY_CDN_HEADER = 'netlify';
```

Reply:
It is removed